Dockerfile.emg

```diff
@@ -0,0 +1,32 @@
+ARG DISTRO_NAME
+ARG DISTRO_VERSION
+FROM $DISTRO_NAME:$DISTRO_VERSION
+
+LABEL maintainer="javier@netmanagers.com.ar"
+
+ARG SALT_INSTALL_METHOD
+ARG SALT_VERSION
+ARG PYTHON_VERSION
+ARG EXTRA_PACKAGES=""
+
+ARG PKGS="udev git net-tools sudo curl openssh-server openssh-clients which findutils glibc-langpack-en $EXTRA_PACKAGES"
+
+RUN echo 'ACCEPT_KEYWORDS="~amd64"' | tee -a /etc/portage/make.conf
+RUN emerge-webrsync \
+ && emerge -u system \
+ && emerge -u world
```

Comment:
I think you should skip updating @system or @world.
We want to test salt and not gentoo and it is a bit of a anti pattern in docker.